src/api/nodeGrouping/types.ts

```diff
@@ -35,6 +40,12 @@ export interface CreateNodeGroupingRuleParams extends IDataSourceParams {
   groupingOptions: NodeGroupingOptions[NodeGroupingType];
 }
 
+export interface UpdateNodeGroupingRuleParams
+  extends Omit<CreateNodeGroupingRuleParams, 'uuid' | 'name'> {
```

Comment:
There is a mismatch between the type declaration and the validation this can cause some bugs.
```suggestion
  extends Partial<Omit<CreateNodeGroupingRuleParams, 'uuid' | 'name'>> {
```